aptos-move/move-examples/token_objects/sources/aptos_token.move

```diff
@@ -30,6 +29,33 @@ module token_objects::aptos_token {
     const ETOKEN_NOT_BURNABLE: u64 = 4;
     /// Attempted to mutate a property map that is not mutable
     const EPROPERTIES_NOT_MUTABLE: u64 = 5;
+    // The collection does not exist
+    const ECOLLECTION_DOES_NOT_EXIST: u64 = 1;
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// Storage state for managing the no-code Collection.
+    struct AptosCollection has key {
+        /// Used to mutate collection fields
+        mutator_ref: Option<collection::MutatorRef>,
+        /// Used to mutate royalties
+        royalty_mutator_ref: Option<royalty::MutatorRef>,
```

Comment:
What happens when both the collection and some token inside that collection have different royalty settings?

Reply:
aptos_token doesn't support adding royalty
but token does and reading royalties is only possible at the token layer... the token will first read itself and then the collection.

Reply:
So any royalty overrides at the token level will take precedence over collection-level royalty?

Reply:
yup, at least if they leverage the on-chain components. off-chain can still result in some amounts of chaos, but this is defined in the spec so it is shame on them if they fail to follow.